modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view.adoc

```diff
@@ -27,6 +28,11 @@ image::odc_customizing_developer_catalog.png[]
 
 [NOTE]
 ====
+As an administrator, you can define which navigation items will display by default for all users. You can also reorder the navigation items.
```

Comment:
```suggestion
As an administrator, you can define the navigation items that appear by default for all users. You can also reorder the navigation items.
```
* IBM Style: "Write in the simple present tense as much as possible."
* IBM Style: "display verb: Use only as a transitive verb. For example, write "the system displays a message" or "the message is displayed" (not "the message displays")."